linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_set_max_life_existing/ansible/shared.yml

```diff
@@ -14,13 +14,13 @@
 {{% if product in ["rhel7", "ol7"] %}}
   ansible.builtin.command:
     cmd: chage -M {{ var_accounts_maximum_age_login_defs }} {{ item }}
-{{% elif product in ["sle12","sle15"] %}}    
+{{% elif product in ["sle12","sle15"] %}}
   ansible.builtin.command:
     cmd:  passwd -q -x {{ var_accounts_maximum_age_login_defs }} {{ item }}
-{{% else %}}        
+{{% else %}}
   ansible.builtin.user:
     user: '{{ item }}'
     password_expire_max: '{{ var_accounts_maximum_age_login_defs }}'
 {{% endif %}}
   with_items: '{{ user_names.stdout_lines }}'
-  when: user_names.stdout_lines | length > 0
+  when: user_names is not skipped and user_names.stdout_lines | length > 0
```

Comment:
`user_names is not skipped` should not be necessary since there is no condition in the task where this register is created. Therefore, it will never be "skipped". Maybe would you like to use `user_names.stdout_lines is defined`?

Reply:
@teacup-on-rockingchair unfortunately I couldn't reproduce any error when executing this Playbook in check mode even without the inclusion of `user_names is not skipped and ...`.

I have tested this in RHEL8 and RHEL9, but I believe it should not be different in SLES. Could you give me mode details on how to reproduce the error expected to be fixed by this change, please?